adapter/adapter.go

```diff
@@ -0,0 +1,21 @@
+package adapter
+
+import "go.uber.org/zap"
+
+type Adapter struct {
```

Comment:
The type `adapter.Adapter` stutters.  It would be better to change this package name to something like `logger`, `logging`, `log`, etc.  There may also be an existing `webpa-common` package that would be appropriate.